src/test/java/org/ethereum/beacon/discovery/NodeRecordTest.java

```diff
@@ -54,7 +53,7 @@ public void testLocalhostV4() throws Exception {
         InetAddress.getByName(expectedHost).getAddress(),
         ((Bytes) nodeRecord.get(EnrField.IP_V4)).toArray());
     assertEquals(expectedUdpPort, nodeRecord.get(EnrField.UDP_V4));
-    assertEquals(expectedTcpPort, nodeRecord.get(EnrField.TCP_V4));
+    //    assertEquals(expectedTcpPort, nodeRecord.get(EnrField.TCP_V4));
```

Comment:
What caused these lines in the tests to be commented out? I can't see how it's related to the logLimit change.

Reply:
Unrelated to this PR, I've removed the TCP-related ports from UDP-based discovery services.